test/library/standard/Reflection/cannotInspectBuiltins.prediff

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh 
+
+# filter out line numbers
+TESTNAME=$1
+OUTFILE=$2
+TMPFILE=$OUTFILE.prediff.tmp
+
+sed -E 's/^.+\.chpl:[0-9]*:/FILE.chpl:n:/' < $OUTFILE > $TMPFILE
+mv $TMPFILE $OUTFILE
+
+sed -E 's/error: .+ is not a class, record, or union type/error: testType is not a class, record, or union type/' < $OUTFILE > $TMPFILE
```

Comment:
I have a slight preference for ensuring we're printing the appropriate type name as part of the error message, but there's very few good reasons we wouldn't be so it's probably okay not to.

That said, I'm not sure what we're saving here besides the number of files checked in - all of these have to happen at compile time, so we're not saving compilation testing time

Reply:
Since there are only 7 distinct types being tested, I can remove this from the prediff and check in 7 good files instead of only 1.

Reply:
That'd be great, thanks!